usl_pipeline/usl_lib/usl_lib/storage/metastore.py

```diff
@@ -266,6 +266,31 @@ def update_scaling_done(
             {"needs_scaling": False, "feature_matrix_path": scaled_feature_matrix_path},
         )
 
+    @classmethod
+    def delete_all_for_study_area(
+        cls,
+        db: firestore.Client,
+        study_area_name: str,
+        page_size: int = 100,
```

Comment:
Do we have any reason to deviate from the default "sensible value set by the API" ?
https://cloud.google.com/python/docs/reference/firestore/latest/google.cloud.firestore_v1.collection.CollectionReference#google_cloud_firestore_v1_collection_CollectionReference_list_documents

Reply:
I think I've read some posts on page_size=20 as a default and it seemed to me to be a bit lower that I would choose. But that's fine. I keep it None by default in order to not override default settings. Nevertheless, I need this parameter to be overridable in integration test to make sure page size does not affect list+delete.